### DISCUSSION TITLE
可否添加md文件导入时的图片同步功能？

### DISCUSSION THREAD

**Discussion by sumingwang233** [windingwind/zotero-better-notes#603] [Ideas]
今天试着用7导入了一个md文件，啪一下很快啊，文字识别都是对的，但是图片显示"unknown"，点开发现，应该是better notes读取图片文件的目录是本地的，而md文件里面是没有图片的，只有一个链接导向。
![image](https://github.com/windingwind/zotero-better-notes/assets/131336715/b9b9f0f4-1071-4817-96d5-4a72f322e9d4)

**Comment by ErPurE** [comment]
同问，我用bn创建的笔记，同步到obsidian之后，关掉之后再打开zotero，里面的图片就变成 unknown了
![image](https://github.com/windingwind/zotero-better-notes/assets/48055565/03a878e8-c143-4bc5-85e5-43d0cc10861a)

**Comment by windingwind** [comment]
支持图片的同步的。ob里面不要用ob的markdown方言来编写图片，用标准markdown或html语法。

**Reply by wbsccs** [reply]
![image](https://github.com/windingwind/zotero-better-notes/assets/95668684/330052be-7aa8-41bf-ac7f-6d6b9d19ea9f)
这应该是标准的markdown语法吧，我尝试好多次都是在ob中对同步笔记进行图片插入后无法在bn中同步出图片

**Comment by windingwind** [comment]
另外功能请求和bug报告请提issue，discussion不便于追踪，我会忘记掉回复。讨论仅用于使用分享，或希望获取其他人意见的话题
